config/env_config.go

```diff
@@ -180,6 +182,12 @@ type EnvConfig struct {
 	//
 	// AWS_S3_USE_ARN_REGION=true
 	S3UseARNRegion *bool
+
+	// Specifies that SDK clients must resolve a dual-stack endpoint for
+	// services.
+	//
+	// AWS_USE_DUALSTACK_ENDPOINT=true
+	UseDualStackEndpoint aws.DualStackEndpoint
```

Comment:
Had similar comment else where. 

IMO `Use + <Option>` seems boolean specific which this is. But since we use enums - may be we should name it agnostic. I think  in general `<option> + State` would be a better pattern to follow. 

`DualStackEndpointState` would be a better name, which can then be set as enabled/disabled or anything else.